storage/tokudb/hatoku_defines.h

```diff
@@ -246,4 +246,22 @@ inline uint tokudb_uint3korr(const uchar *a) {
     return uint3korr(b);
 }
 
-#endif // _HATOKU_DEFINES_H
+typedef unsigned int pfs_key_t;
```

Comment:
PerconaFT has it's own declaration in the header file which is supposed to be internal library header.